cli/tests/data/package/create-directory/marathon.json.mustache

```diff
@@ -5,8 +5,8 @@
   "cpus": {{cassandra.framework.cpus}},
   "mem": {{cassandra.framework.mem}},
   "uris": [
-    "{{package.assets.uris.cassandra-mesos.tar.gz}}",
-    "{{package.assets.uris.jre-linux.tar.gz}}"
+    "{{resource.assets.uris.cassandra-mesos}}",
+    "{{resource.assets.uris.jre-linux}}"
```

Comment:
I thought was agreed that we would turn this to:

```
{{resource.assets.uris.cassandra-mesos-tar-gz}}
{{resource.assets.uris.jre-linux-tar-gz}}
```

I.e. replace `.` with `-`.